app/services/blacklight/search_service.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+# SearchService returns search results from the repository
+module Blacklight
+  class SearchService
+    include Blacklight::RequestBuilders
+
+    def initialize(blacklight_config)
+      @blacklight_config = blacklight_config
+    end
+
+    # TODO: Can this be private?
```

Comment:
👌 Does it make sense to add that comment to the TODO, or do you think it's a near-term sort of change?